pyproject.toml

```diff
@@ -18,13 +18,14 @@ maintainers = [
     { name = "O.N. Ebbens", email = "o.ebbens@artesia-water.nl" },
     { name = "M.A. Vonk", email = "vonk.mart@gmail.com" },
 ]
-requires-python = ">= 3.9"
+requires-python = ">= 3.10"
 dependencies = [
     "numpy >= 1.22.4",
     "matplotlib >= 3.6.1",
     "pandas >= 2.1",
     "scipy >= 1.8.1, !=1.13.0",
-    "numba >= 0.55.2",
+    "numba >= 0.55.2, < 0.61.0 ; python_version < '3.13'",
+    "numba >= 0.61.0rc1 ; python_version >= '3.13'",
```

Comment:
We could replace 0.61.rc1 with 0.61.rc2 since there were some technical issues with rc1 in [llvmlite](https://numba.discourse.group/t/ann-numba-0-61-0rc1-and-llvmlite-0-44-0rc1-are-out/2816)